hc/accounts/tests/test_profile.py

```diff
@@ -35,7 +35,7 @@ def test_it_sends_report(self):
         message = mail.outbox[0]
 
         self.assertEqual(message.subject, 'Monthly Report')
-        self.assertIn("This is a monthly report sent by healthchecks.io", message.body)
+        self.assertIn("report sent by healthchecks.io", message.body)
```

Comment:
You could still use the old format and have it state the frequency

Reply:
I will look at how to implement that